server/app/views/applicant/ProgramIndexView.java

```diff
@@ -180,12 +183,61 @@ private DivTag mainContent(
                 Math.max(relevantPrograms.unapplied().size(), relevantPrograms.submitted().size()),
                 relevantPrograms.inProgress().size()));
 
+    if (featureFlags.isIntakeFormEnabled(request)
+        && relevantPrograms.commonIntakeForm().isPresent()) {
+      DivTag findBenefitsSection = div().withClass(ReferenceClasses.APPLICATION_PROGRAM_SECTION);
```

Comment:
Would it make sense to pull this out into a `renderIntakeFormComponent()`-like function? Or is that a premature refactor?